### ISSUE TITLE
auction_succeeded call

### ISSUE THREAD

**Issue by GopherJ** [parallel-finance/parallel#1037]

**Comment by GopherJ** [comment]
not sure if this is necessary to have

**Comment by GopherJ** [comment]
no need to have this for now

**Comment by GopherJ** [comment]
this can be useful for frontend

**Comment by GopherJ** [comment]
@drbh can you take this. We should have a succeed phase between `Closed` and `Expired` so that the frontend can know this parachain won the auction.

`slot_expired` will then need to check:

```
ensure!(vault.phase == VaultPhase::Expired, Error::<T>::IncorrectPhase);
```

this change will be included in `v1.7.5-rc1` together with @ashi20192020ayush 's PR
